pgsql/expected/pointcloud.out

```diff
@@ -450,14 +437,12 @@ SELECT PC_AsText(PC_Patch(ARRAY[PC_MakePoint(3, ARRAY[-127, 45, 124.0, 4.0])]::p
 (1 row)
 
 SELECT 'pc_id1', PC_PCId(PC_Patch(PC_MakePoint(3, ARRAY[-1,-2,-3,-4])));
-WARNING:  Value -4 truncated to 0 to fit in uint16_t
```

Comment:
@mbredif it looks like I had to revert https://github.com/LI3DS/pointcloud/pull/60/commits/835c6999246c4ec77b40c54e88c03d9f71a4d112. Why did you add this commit? Was it required in your case?